### PR TITLE
[SPARK-2827][GraphX] Add collectDegreeDist to compute the distribute of vertex degrees in GraphOps

### PR DESCRIPTION
**Pull request by maropu**
This is from #4399.

### CODE REVIEW

**Comment by maropu** [comment]
@andrewor14 Done. We'll wait for tests done.

**Comment by hvanhovell** [comment]
retest this please

**Comment by maropu** [comment]
retest this please

**Comment by hvanhovell** [comment]
@maropu you might need to merge the latest master. My PR (https://github.com/apache/spark/pull/10509) non-deterministically broke the build, and has since been reverted. Sorry about that.

**Comment by maropu** [comment]
@hvanhovell Ah, okay. I'll rebase this pr.

**Comment by maropu** [comment]
retest this please

**Comment by maropu** [comment]
@andrewor14 ISTM it's ready to merge.

**Comment by maropu** [comment]
@andrewor14 ping

**Comment by maropu** [comment]
@andrewor14 ping

**Comment by maropu** [comment]
@andrewor14 ping

**Comment by andrewor14** [comment]
@maropu I actually do not maintain GraphX code. You need to ping @ankurdave instead.

File: graphx/src/main/scala/org/apache/spark/graphx/GraphOps.scala [hunk 1]
```diff
@@ -17,6 +17,8 @@
 
 package org.apache.spark.graphx
 
+import org.apache.spark.graphx.util.FrequencyDistribution
```
Comment (andrewor14) [inline graphx/src/main/scala/org/apache/spark/graphx/GraphOps.scala]: by the way this import order is wrong. It should be below the scala imports

**Comment by maropu** [comment]
@andrewor14 okay cc: @ankurdave

**Comment by maropu** [comment]
@ankurdave ping

**Comment by maropu** [comment]
I think the GraphX features seems freeze, so I'll close this.

**Pull request closed without merging**